src/twisted/topfiles/8900.feature

```diff
@@ -0,0 +1,2 @@
+Added twisted.internet.defer.extractCurrentResult function to enable extracting
```

Comment:
The style guide for news fragments suggests "The new function twisted.internet.defer.extractCurrentResult ..."